examples/miscellaneous.xml

```diff
@@ -63,6 +63,13 @@
     <!-- Example 6 -->
     <Schema Namespace="org.example" Alias="self">
       <Annotation Term="Core.Description" String="Example schema" />
+      <Annotation Term="Core.Version">
+        <Add>
+          <Int>1</Int>
+          <Annotation Term="Core.Description" String="add major and minor part" />

```

Comment:
This `Annotation` between the first and second child of `Add` is not allowed by the `xs:sequence` given in https://github.com/oasis-tcs/odata-csdl-schemas/blob/e5b9fc7bc260bec8fdbba48fd4ddee1a84277981/schemas/edm.xsd#L420-L426

Reply:
Which is not a problem for me if it is possible to put annotations before and after the operand expressions.

Reply:
But the [OData-CSDL-XML] spec does not restrict the ordering of XML elements in any way.